app/helpers/compute_resources_vms_helper.rb

```diff
@@ -294,4 +294,10 @@ def vm_console_action(vm)
       }
     )
   end
+
+  def vmware_vm_hypervisor_name(vm)
+    vm.hypervisor&.name rescue nil
```

Comment:
Why the double rescue? :-)
Can we just catch the proper exception and not everything?

Reply:
Just messy me.. :man_facepalming:

Reply:
Corrected.